### PR TITLE
{bio}[foss/2016b,intel/2016b] PheWAS v0.12

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 9.3, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/6d273e33a322cd407f704724b1374809 for a full test report.

**Comment by vanzod** [comment]
Depends on #5773

File: easybuild/easyconfigs/p/PheWAS/PheWAS-0.12-foss-2016b-R-3.3.3.eb [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+dependencies = [
+    ('R', '3.3.3', '-X11-20160819'),
+]
+
+exts_defaultclass = 'RPackage'
+exts_list = [
+    ('meta', '4.9-0', {
+        'source_tmpl': '%(name)s_%(version)s.tar.gz',
+        'source_urls': ['http://cran.r-project.org/src/contrib/'],
```
Comment (boegel) [inline easybuild/easyconfigs/p/PheWAS/PheWAS-0.12-foss-2016b-R-3.3.3.eb]: @vanzod You'll also need to include `https://cran.r-project.org/src/contrib/Archive/meta/` here (same goes for other extensions)...

Or, you can specify default extension options with the general `exts_default_options` easyconfig parameter, and only override where needed:

```python
exts_default_options = {
    'source_urls': [
        'http://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
        'http://cran.r-project.org/src/contrib/',  # current version of packages
        'http://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
    ],
    'source_tmpl': '%(name)s_%(version)s.tar.gz',
}
```

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 9.4, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/57afe3df8fe98e0ce1f63ad92044199c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
gligar03.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3f8c0d65530411d1b4ce91a8ffeebba2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 69 out of 69 (2 easyconfigs in this PR)
node2672.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/cf8b6e7875e3bb665b87ad4d3dc8c4c3 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**